server/election/lease.go

```diff
@@ -11,39 +11,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package member
+package election
 
 import (
 	"context"
 	"sync/atomic"
 	"time"
 
 	"github.com/pingcap/log"
+	"github.com/pingcap/pd/v4/pkg/etcdutil"
 	"github.com/pkg/errors"
 	"go.etcd.io/etcd/clientv3"
 	"go.uber.org/zap"
 )
 
-// LeaderLease is used for renewing leadership of PD server.
-type LeaderLease struct {
-	client       *clientv3.Client
-	lease        clientv3.Lease
-	ID           clientv3.LeaseID
-	leaseTimeout time.Duration
+const (
+	revokeLeaseTimeout = time.Second
+	requestTimeout     = etcdutil.DefaultRequestTimeout
+	slowRequestTime    = etcdutil.DefaultSlowRequestTime
+)
 
-	expireTime atomic.Value
+// Lease is used for campaigning and renewing elected leadership.
+type Lease struct {
+	// purpose is used to show what this election for
+	Purpose string
+	// etcd client and lease
+	client *clientv3.Client
+	lease  clientv3.Lease
+	ID     clientv3.LeaseID
+	// lifetime related fields
```

Comment:
I think the comment here could be more detailed as I didn't get the point for this comment.